backend/tracim_backend/models/data.py

```diff
@@ -1163,6 +1163,7 @@ def children(self) -> ["Content"]:
             .query(func.max(ContentRevisionRO.revision_id))
             .filter(ContentRevisionRO.content_id.in_(children_content_ids))
             .group_by(ContentRevisionRO.content_id)
+            .order_by(ContentRevisionRO.content_id)
```

Comment:
it should be content_revision_id, doesn't it ?

Reply:
not here. we sort content not revision here.